tests/unit/ui/components/TestRating.js

```diff
@@ -228,7 +239,7 @@ describe(__filename, () => {
   });
 
   it('selects stars on hover', () => {
-    const root = render();
+    const root = render({ loading: false });
```

Comment:
You mostly use `renderWithRating()` to end up with a component that isn't in a loading state in these tests (including in "does nothing when you hover over stars", below), but for these two tests you use `render({ loading: false })`. Is there a reason for that difference in these two tests?